PKGBUILD

```diff
@@ -1,19 +1,16 @@
 
-pkgname="libixp-hg"
+pkgname="libixp"
 pkgver=131
 pkgrel=1
-pkgdesc="The latest hg pull of libixp, a simple 9P filesystem library"
-url="http://libs.suckless.org/libixp"
+pkgdesc="a simple 9P filesystem library"
+url="http://github.com/0intro/libixp"
 license=(MIT)
 arch=(i686 x86_64)
-makedepends=(mercurial)
```

Comment:
Shouldn't we add git here?

Reply:
git would only be needed there if we wanted to build the package from head, which I don't think we should do.

here is my unsolicited rant on the matter;
I'd rather have tagged versions of the package so the package managers can update the package automatically.  And [publishing](https://fusion809.github.io/how-to-create-archlinux-repository/#github) this as a package doesn't seem that hard

the entire idea of building packages I use day to day from head/tip makes me uneasy, moreso when packages depend on other packages that are built from head/tip.

I understand the use case for it, for instance in development scenarios it'd be very useful, and I'm happy to go back to the head/tip paragidm (less work :smile: ) but I'd rather have stability of a released version.

What do you think?

Reply:
I'm OK to distribute GitHub releases, but I don't feel it would be worth naming the releases "x.y.z". Would it be fine to have releases named by a date, like 20180323?

Reply:
that seems totally fine :+1:  https://wiki.archlinux.org/index.php/PKGBUILD#pkgver

Reply:
Oh hello, I'll gladly pick this up and put it in AUR :)

Reply:
Go for it Daniel!

Reply:
apparently, someone has already done it https://aur.archlinux.org/packages/libixp-git :D